website/.eslintrc

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
Since there's an eslintrc in the parent dir, this might not play nice with IDE plugins.  

Potential workaround: Use your IDE to open only the `website/` directory.

For CI, i had GitHub Actions just temporarily remove the parent dir's eslintrc file while the deploy script runs to circumvent the problem.  Not a good solution for everyday development though, so if anyone gets stuck trying to get this to work when contributing to the Fleet website, make an issue and I'll help you troubleshoot

cc @noahtalerman